tests/testthat/test-col_identify.R

```diff
@@ -0,0 +1,46 @@
+test_that("col_identify should return a datafame", {
+
+  mpg <- system.file("extdata", "mpg.csv", package="collinearityR")
```

Comment:
mtcars is a R built-in data which can be used directly. Try mpg <- mtcars. It is smaller than the csv file but it should be enough for the testing purpose?